plugin/AL_USDMayaTestPlugin/AL/usdmaya/fileio/import_instances.cpp

```diff
@@ -65,7 +65,7 @@ TEST(import_instances, merged)
   {
     auto it = expectedTransforms.find(xformResults[i].asChar());
     ASSERT_TRUE(it != expectedTransforms.end());
-    expectedTransforms.erase(it);
+    expectedTransforms.erase(*it);
```

Comment:
If you are now erasing by key, can't you just do:
```c++
expectedTransforms.erase(xformResults[i].asChar());
```
(Saves the cost of finding the key twice)

Reply:
You're right. Hopefully this second commit addresses this.